src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).orElseThrow(() ->
+                new DataProcessingException("Can`t get from DB by id" + id));

```

Comment:
```suggestion
    public Car get(Long id) {
        return carDao.get(id).orElseThrow();
```
це ніби ексепшн для ДАО